src/app/api/Mock/MockApi.ts

```diff
@@ -15,35 +15,40 @@ import {
     assignments,
     training,
     courthouses,
-    WORK_SECTIONS,
     COURTROOMS,
     courtrooms,
     regions,
     assignmentDuties,
-    JAIL_ROLES
+    RUNS,
+    JAIL_ROLES,
+    ALTERNATE_ASSIGNMENTS
 } from "./MockData";
+import {
+     isCourtAssignment,
+     isJailAssignment, 
+     isEscortAssignment,
+     isOtherAssignment
+} from '../utils';
 import { randomDelay } from "../PromiseExtensions";
 
 
 // Helpers
 function getAssignmentTitle(assignment: Partial<Assignment>): string {
     let assignmentTitle = "Assignment Title";
 
-    if (assignment.workSectionId) {
-        assignmentTitle = WORK_SECTIONS[assignment.workSectionId];
-        switch (WORK_SECTIONS[assignment.workSectionId]) {
-            case WORK_SECTIONS.COURTS:
-                if (assignment.location && assignment.location.courtroomId) {
-                    assignmentTitle = COURTROOMS[assignment.location.courtroomId];
-                }
-                break;
-            case WORK_SECTIONS.JAIL:
-                if (assignment.extraDetails && assignment.extraDetails.jailRoleId) {
-                    assignmentTitle = JAIL_ROLES[assignment.extraDetails.jailRoleId]
-                }
-                break;
-        }
+    if(isCourtAssignment(assignment)){
```

Comment:
I really like how this looks now.  Also pretty nice that this got pulled out into its own method 👍